checks/raw/fuzzing.go

```diff
@@ -85,6 +85,26 @@ var languageFuzzSpecs = map[clients.LanguageName]languageFuzzConfig{
 		Name:        fuzzers.PropertyBasedHaskell,
 		Desc:        propertyBasedDescription("Haskell"),
 	},
+
+	// Fuzz patterns for Elixir based on property-based testing.
+	clients.Elixir: {
+		filePatterns: []string{"*.ex", "*.exs"},
+		// Look for direct imports of PropCheck, and StreamData.
+		funcPattern: `use\s+(PropCheck|ExUnitProperties)`,
+		Name:        fuzzers.PropertyBasedElixir,
+		Desc:        propertyBasedDescription("Elixir"),
+	},
+
+	// Fuzz patterns for Gleam based on property-based testing.
+	clients.Gleam: {
+		filePatterns: []string{"*.ex", "*.exs", "*.erl", "*.hrl"},
+		// Look for direct imports of PropCheck, and StreamData.
+		funcPattern: `(use\s+(PropCheck|ExUnitProperties)|` + // Elixir libraries
+			`-include_lib\("(eqc|proper)/include/(eqc|proper).hrl"\)\.|` + // Erlang libraries
+			`import\s+qcheck)`, // Gleam library
```

Comment:
I'm unfamiliar with Gleam. Should it not be looking at a `.gleam` files?

Reply:
Meaning, yes, I forgot to add `*.gleam` but Gleam also allows calling Erlang and Elixir files from within Gleam files.
I think @lpil and @giacomocavalieri can confirm this :)

Reply:
And that wouldn't be covered by the existing Erlang/Elixir fuzz patterns? I worry these patterns will drift out of date if someone updates the patterns for Erlang or Elixir and forgets to update them here.

Scorecard will look at multiple languages in the repository (as defined by languages with more than 1/8th of the LOC in a repo). So I guess it depends on the ratio of files in a repo for this to matter or not?

Reply:
Oh, you are right. If in a Gleam project scorecard detects Erlang or Elixir, then those will be recognise.
Absolutely right. I will fix this

Reply:
I think this push should fix it. Thanks again, I somehow was thinking about all the combinations the language can do, instead of focusing on the files scorecard recognises. Thanks again!

Reply:
> And that wouldn't be covered by the existing Erlang/Elixir fuzz patterns? I worry these patterns will drift out of date if someone updates the patterns for Erlang or Elixir and forgets to update them here.
> 
> Scorecard will look at multiple languages in the repository (as defined by languages with more than 1/(4*N)th of the LOC in a repo (where N = num langs). So I guess it depends on the ratio of files in a repo for this to matter or not?

Yes, it may depend. Maybe it makes sense to leave it for now specific to Gleam files, and if Gleam relies too much on Erlang or Elixir for property-based testing, maybe someone creates a new PR in that case?

It feels like, given that Gleam has its own property-based library, maybe the Gleam community uses that and does not rely that much on Erlang/Elixir for those tests.

Does this sound reasonable?

Reply:
A Gleam project maybe include a small amount of JavaScript or Erlang, but typically there is none.

It's similar to C in Python libraries.

Reply:
Great, then this PR should be fine for most Gleam projects.
Thanks for the quick response rate @lpil  and @spencerschrock